### PR TITLE
feat: improve orphan build reviewed status

### PR DESCRIPTION
**Pull request by gregberge**

### CODE REVIEW

File: apps/frontend/src/pages/Build/BuildWorkspace.tsx [hunk 1]
```diff
@@ -108,7 +109,7 @@ export const BuildWorkspace = (props: {
               build && (
                 <>
                   <BuildDetail build={build} repoUrl={repoUrl} />
-                  {build.type === "orphan" && (
+                  {build.type === "orphan" && build.status === BuildStatus.DiffDetected && (
```
Comment (jsfez) [inline apps/frontend/src/pages/Build/BuildWorkspace.tsx:112]: I don't understand this line. Can you explain why did you add this?

Comment (gregberge) [reply apps/frontend/src/pages/Build/BuildWorkspace.tsx:112]: If already reviewed, no dialog.

Comment (jsfez) [reply apps/frontend/src/pages/Build/BuildWorkspace.tsx:112]: DiffDetected !== already reviewed

Comment (gregberge) [reply apps/frontend/src/pages/Build/BuildWorkspace.tsx:112]: That's why it's a === and not a !==
